test/over_react/component_declaration/builder_integration_tests/component2/required_accessor_integration_test.dart

```diff
@@ -256,11 +256,10 @@ class _$ComponentTestProps extends UiProps {
 class ComponentTestComponent extends UiComponent2<ComponentTestProps> {
   @override
   get propTypes => {
-      getPropKey((props) => props.requiredAndLengthLimited, typedPropsFactory):
-          (props, propName, componentName, location, propFullName) {
+      getPropKey((p) => p.requiredAndLengthLimited): (props, info) {
         final length = props.requiredAndLengthLimited?.length;
         if (length != 2) {
-          return PropError.value(length, propName, 'must have a length of 2');
+          return PropError.value(length, info.propName, 'must have a length of 2');
         }
         return null;
```

Comment:
`return null` is no longer part of the boilerplate
```suggestion
```

Reply:
https://github.com/cleandart/react-dart/pull/228#discussion_r343752079